homeassistant/components/lifx/__init__.py

```diff
@@ -20,7 +21,10 @@
 )
 
 CONFIG_SCHEMA = vol.Schema(
-    {DOMAIN: {LIGHT_DOMAIN: vol.Schema(vol.All(cv.ensure_list, [INTERFACE_SCHEMA]))}},
+    {
+        DOMAIN: {LIGHT_DOMAIN: vol.Schema(vol.All(cv.ensure_list, [INTERFACE_SCHEMA]))},
+        vol.Optional(CONF_DEFAULT_TRANSITION): cv.positive_float,
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

There is the options flow interface to set options for a config entry:
https://developers.home-assistant.io/docs/config_entries_options_flow_handler

Reply:
Ah fair enough, wasn't sure how the change applied to existing integrations or not. Will RE-raise the PR's if I reimplement it with the config flow